bugwarrior/services/gitlab.py

```diff
@@ -327,11 +327,11 @@ def filter_repos(self, repo):
         return is_included
 
     def _get_notes(self, rid, issue_type, issueid):
-        tmpl = '{scheme}://{host}/api/v3/projects/%d/%s/%d/notes' % (rid, issue_type, issueid)
+        tmpl = '{scheme}://{host}/api/v4/projects/%d/%s/%d/notes' % (rid, issue_type, issueid)
         return self._fetch_paged(tmpl)
 
     def annotations(self, repo, url, issue_type, issue, issue_obj):
-        notes = self._get_notes(repo['id'], issue_type, issue['id'])
+        notes = self._get_notes(repo['id'], issue_type, issue['iid'])
```

Comment:
Typo?

Reply:
It's a gitlab API change from v3 to v4. When query something belong to merge request or issue, the API was using a site-wide id (the `id` property) to referring MR/issue, in v4 they change it to project-wide id (the `iid` property)